docs/spec/sources.md

```diff
@@ -345,3 +345,17 @@ this instantiation brings onto the eventing mesh. For a more detailed
 description, please refer to the
 [Event Registry](https://knative.dev/docs/eventing/event-registry/)
 documentation.
+
+## Source Event delivery
+
+Sources MUST produce CloudEvents. The output MUST be via the HTTP binding specified in
```

Comment:
`The output MUST be via the HTTP binding specified` I am not sold this is a MUST.

Reply:
it is the HTTP part that I am questioning.